src/vs/workbench/api/node/extHostApiCommands.ts

```diff
@@ -86,7 +86,8 @@ class ExtHostApiCommands {
 			description: 'Execute signature help provider.',
 			args: [
 				{ name: 'uri', description: 'Uri of a text document', constraint: URI },
-				{ name: 'position', description: 'Position in a text document', constraint: types.Position }
+				{ name: 'position', description: 'Position in a text document', constraint: types.Position },
+				{ name: 'triggerCharacter', description: 'Trigger signature help when the user types the character, like `,` or `(`', constraint: String }
 			],
```

Comment:
this is an array and called triggerCharacters, also it's optional which I don't know if we express yet